src/unix/notbsd/linux/mod.rs

```diff
@@ -728,6 +728,7 @@ extern {
 
 cfg_if! {
     if #[cfg(any(target_env = "musl",
+                 target_os = "fuchsia",
```

Comment:
This is fine for now, but if the two start to diverge we can just copy musl over and update fuschia as necessary